devito/types/grid.py

```diff
@@ -305,7 +310,7 @@ def shape_local(self):
         return self._distributor.shape
 
     @property
-    def dimension_map(self):
+    def size_map(self):
```

Comment:
you're changing public API here

perhaps leave

```
from logging import deprecation
...
@property
def dimension_map(self):
     deprecation(...)
     return self.size_map
```

Though honestly unlikely to be used on the outside......

Reply:
Ah, I used `dimension_map` for something else since iirc, this was an otherwise unused property with a name which was not terribly descriptive

Reply:
That's probably acceptable, I don't see users accessing unadvertised APIs ?